### PR TITLE
guidelines: update list of authors in Chapter 3

### PR DESCRIPTION
**Pull request by riedde**
This PR is a proposal by @music-encoding/ig-metadata
The authors of the texts are referenced by the `Co-authored-by` keyword.

_Please note:_ This PR is going to be merged into the branch `ig-metadata`!

Reviews and constructive suggestions are welcome!

### CODE REVIEW

**Review by bwbohl — APPROVED** [review]

File: source/docs/03-metadata.xml [hunk 1]
```diff
@@ -39,21 +39,26 @@
                <name ref="https://api.github.com/users/fujinaga">Ichiro Fujinaga</name>
                <name ref="https://api.github.com/users/axgeertinger">Axel Geertinger</name>
                <name ref="https://api.github.com/users/ahankinson">Andrew Hankinson</name>
-               <name ref="https://api.github.com/users/irmlindcapelle">irmlindcapelle</name>
+               <name ref="https://api.github.com/users/irmlindcapelle">Irmlind Capelle</name>
```
Comment (musicEnfanthen) [inline source/docs/03-metadata.xml]:
```suggestion
               <name ref="https://api.github.com/users/irmlindcapelle">irmlindcapelle</name>
```

File: source/docs/03-metadata.xml [hunk 2]
```diff
@@ -39,21 +39,26 @@
...
+               <name ref="https://api.github.com/users/gucl-mu">gucl-mu</name>
                <name ref="https://api.github.com/users/frakel">franz kelnreiter</name>
                <name ref="https://api.github.com/users/kepper">kepper</name>
                <name ref="https://api.github.com/users/zolaemil">Zoltan Komives</name>
                <name ref="https://api.github.com/users/DDMAL-LabManager">DDMAL LabManager</name>
                <name ref="https://api.github.com/users/uliska">Urs Liska</name>
                <name ref="https://api.github.com/users/elsinhadl">Elsa De Luca</name>
                <name ref="https://api.github.com/users/napulen">Néstor Nápoles López</name>
-               <name ref="https://api.github.com/users/MajaHartwig">MajaHartwig</name>
+               <name ref="https://api.github.com/users/MajaHartwig">Maja Hartwig</name>
```
Comment (musicEnfanthen) [inline source/docs/03-metadata.xml]:
```suggestion
               <name ref="https://api.github.com/users/MajaHartwig">MajaHartwig</name>
```

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]
Use Github name if no real name is provided via GH API

**Comment by ahankinson** [comment]
Is there a reason the GitHub API URI is preferred over ORCID or VIAF?

File: source/docs/03-metadata.xml [hunk 3]
```diff
@@ -40,6 +40,8 @@
                <name ref="https://api.github.com/users/axgeertinger">Axel Geertinger</name>
                <name ref="https://api.github.com/users/ahankinson">Andrew Hankinson</name>
                <name ref="https://api.github.com/users/irmlindcapelle">irmlindcapelle</name>
+               <name ref="https://api.github.com/users/doerners">Sophia Dörner</name>
```
Comment (musicEnfanthen) [inline source/docs/03-metadata.xml]: Move "Dörner" above "Dubowy"

File: source/docs/03-metadata.xml [hunk 4]
```diff
@@ -40,6 +40,8 @@
...
+               <name ref="https://api.github.com/users/gucl-mu">gucl-mu</name>
```
Comment (musicEnfanthen) [inline source/docs/03-metadata.xml]: Move "gucl-mu" above "Hankinson"

**Comment by musicEnfanthen** [comment]
@ahankinson At the moment, it is consistent with only GitHub API. But yes, we should discuss with the @music-encoding/guidelines-editors if we want to be limited to those in the future.

File: source/docs/03-metadata.xml
Comment (musicEnfanthen) [inline source/docs/03-metadata.xml:1]: @riedde I cannot push directly to this PR, so please change the list to the following:

```
               <name ref="https://api.github.com/users/bwbohl">Benjamin W. Bohl</name>
               <name ref="https://api.github.com/users/BruxDDay">BruxDDay</name>
               <name ref="https://api.github.com/users/elsinhadl">Elsa De Luca</name>
               <name ref="https://api.github.com/users/dinamix">dinamix</name>
               <name ref="https://api.github.com/users/doerners">Sophia Dörner</name>
               <name ref="https://api.github.com/users/ndubo">Norbert Dubowy</name>
               <name ref="https://api.github.com/users/fujinaga">Ichiro Fujinaga</name>
               <name ref="https://api.github.com/users/axgeertinger">Axel Geertinger</name>
               <name ref="https://api.github.com/users/gucl-mu">gucl-mu</name>
               <name ref="https://api.github.com/users/ahankinson">Andrew Hankinson</name>
               <name ref="https://api.github.com/users/irmlindcapelle">irmlindcapelle</name>
               <name ref="https://api.github.com/users/frakel">franz kelnreiter</name>
               <name ref="https://api.github.com/users/kepper">kepper</name>
               <name ref="https://api.github.com/users/zolaemil">Zoltan Komives</name>
               <name ref="https://api.github.com/users/DDMAL-LabManager">DDMAL LabManager</name>
               <name ref="https://api.github.com/users/uliska">Urs Liska</name>
               <name ref="https://api.github.com/users/napulen">Néstor Nápoles López</name>
               <name ref="https://api.github.com/users/MajaHartwig">MajaHartwig</name>
               <name ref="https://api.github.com/users/musicEnfanthen">Stefan Münnich</name>
               <name ref="https://api.github.com/users/pe-ro">pe-ro</name>
               <name ref="https://api.github.com/users/lpugin">Laurent Pugin</name>
               <name ref="https://api.github.com/users/JRegimbal">Juliette Regimbal</name>
               <name ref="https://api.github.com/users/rettinghaus">Klaus Rettinghaus</name>
               <name ref="https://api.github.com/users/riedde">Dennis Ried</name>
               <name ref="https://api.github.com/users/mss2221">Mark Saccomano</name>
               <name ref="https://api.github.com/users/aseipelt">Agnes Seipelt</name>
               <name ref="https://api.github.com/users/margrethesb">Margrethe Støkken Bue</name>
               <name ref="https://api.github.com/users/martha-thomae">Martha E. Thomae</name>
               <name ref="https://api.github.com/users/raffazizzi">Raffaele Viglianti</name>
               <name ref="https://api.github.com/users/vigliensoni">Gabriel Vigliensoni</name>
               <name ref="https://api.github.com/users/th-we">Thomas Weber</name>
               <name ref="https://api.github.com/users/musicog">David M. Weigl</name>
```

Comment (riedde) [reply source/docs/03-metadata.xml:1]: Done on https://github.com/music-encoding/music-encoding/pull/1093/commits/70870ac7e4be9743d5015f88a57a64e5c7e39b0c

**Review by musicEnfanthen — APPROVED** [review]

**Pull request merged**